bin/console

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ruby
+
+require 'bundler/setup'
+require 'unleash/incognia'
+
+# You can add fixtures and/or initialization code here to make experimenting
+# with your gem easier. You can also use a different console, if you like.
+
+# (If you use this, don't forget to add pry to your Gemfile!)
+# require 'pry'
+# Pry.start
```

Comment:
:eyes: why not?

Reply:
:joy: 

Done: 955913d5e0e33678a2d9d40ed4f6824f12944d0f

Reply:
:juliana-glad: